python-packages/contract_wrappers/test/test_exchange_wrapper.py

```diff
@@ -0,0 +1,122 @@
+"""Test 0x Exchnage wrapper."""
+import random
+import pytest
```

Comment:
add a blank line here, to separate standard library imports from 3rd party imports.

Reply:
done.